package.json

```diff
@@ -289,6 +289,7 @@
     "eslint-plugin-you-dont-need-lodash-underscore": "^6.14.0",
     "html-webpack-plugin": "^5.5.0",
     "http-server": "^14.1.1",
+    "husky": "^9.1.5",
```

Comment:
Out of curiosity, why did we need to reintroduce this package for just changing the order of a couple emojis? This was the package that modified everyone's git hooks.

Reply:
We did not need to and this shouldn't have been merged. I was getting the error from the git hooks so I added the package and the change accidentally got pushed 😬